main/2021/CG-05-25.md

```diff
@@ -25,6 +25,7 @@ Installation is required, see the calendar invite.
 1. Adoption of the agenda
 1. Proposals and discussions
     1. Review of action items from prior meeting.
+    1. Present [WebAssembly, UTF-8 and Web Platform APIs - Or: Please don't break the Web 🥺](https://youtu.be/IhvwO0zkbXU) (Daniel Wirtz) [15 mins]
```

Comment:
The video appears to be "private"

Reply:
I haven't made it public yet. I don't see a reason why I couldn't make it public ahead of time, so everyone can already watch when they have time etc. Would know about scheduling first, though, in case I have to edit.